### PR TITLE
Move methods to src

### PR DESCRIPTION
**Pull request by ganes1410**
thsi pull request regarding #1 
move all methods to src.Also changed the test path.
All the tests passed.
screenshot from 2018-03-10 18-44-06

### CODE REVIEW

**Comment by WaldoJeffers** [comment]
@ganes1410 Thank you a lot for this! This made me realize I should add configuration files for editor's preferences & tools such as Prettier to avoid formatting issues (tabs vs spaces etc...).

However, some files were correctly renamed without changes, while others were deleted & created as new files (which loses the file's Git history) .
image
You can use `git mv old_name new_name` to do so (some text editors do this, like Atom). I'd be glad to accept your PR with this change 😄

File: package-lock.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "async",
+  "name": "conductor",
```
Comment (WaldoJeffers) [inline package-lock.json]: Excellent catch 👍

File: src/awaitAll.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+const valueIterator = require('./valueIterator')
+const iterate = require('./iterate')
+const values = require('./values')
+const dump = require('./dump')
+const mapTransformer = require('../transformers/map')
```
Comment (WaldoJeffers) [inline src/awaitAll.js]: Actually, I'd like to move *all* methods (internal, transformers) to the `src` folder if that's OK with you

File: src/filter.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
+const curry = require('./curry')
+const filterTransformer = require('../transformers/filter')
+const getAddReducer = require('../internal/getAddReducer')
+const getSeed = require('../internal/getSeed')
```
Comment (WaldoJeffers) [inline src/filter.js]: Actually, I'd like to move *all* methods (internal, transformers) to the `src` folder if that's OK with you

File: src/into.js [hunk 4]
```diff
@@ -1,8 +1,8 @@
 const curry = require('./curry')
 const transduce = require('./transduce')
-const getAddReducer = require('./internal/getAddReducer')
+const getAddReducer = require('../internal/getAddReducer')
```
Comment (WaldoJeffers) [inline src/into.js]: Actually, I'd like to move *all* methods (internal, transformers) to the `src` folder if that's OK with you

File: src/map.js [hunk 5]
```diff
@@ -3,13 +3,13 @@ const isPromise = require('./isPromise')
 const some = require('./some')
 const ifElse = require('./ifElse')
 const compose = require('./compose')
-const mapTransformer = require('./transformers/map')
+const mapTransformer = require('../transformers/map')
```
Comment (WaldoJeffers) [inline src/map.js]: Actually, I'd like to move *all* methods (internal, transformers) to the `src` folder if that's OK with you

**Comment by ganes1410** [comment]
there are so many changes.can i delete this pull request and make a new one

**Comment by WaldoJeffers** [comment]
Sure thing @ganes1410 , if that's not too much trouble for you

**Comment by ganes1410** [comment]
thank you @WaldoJeffers

**Pull request closed without merging**